src/test/kotlin/net/maizegenetics/phgv2/utils/SeqUtilsTest.kt

```diff
@@ -0,0 +1,234 @@
+package net.maizegenetics.phgv2.utils
+
+import biokotlin.genome.fastaToNucSeq
+import com.github.ajalt.clikt.testing.test
+import net.maizegenetics.phgv2.cli.AgcCompress
+import net.maizegenetics.phgv2.cli.TestExtension
+import net.maizegenetics.phgv2.cli.TestExtension.Companion.testOutputFastaDir
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeAll
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.Paths
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+import org.junit.jupiter.api.assertThrows
+
+class SeqUtilsTest {
+
+    companion object {
```

Comment:
You might look at AlignAssembliesTest.  I have @ExtendWith(TestExtension::class) above the class definition, and I don't need any of this?

Reply:
@tcasstevens I think the difference is you don't have the setup and teardown.  I need those to create the assemblies.agc file prior to running any tests that retrieve data.

I'm going to merge this branch, but keep it open so I can finish up with the non-contig AGC retrieval commands.  I'll look at it again then.